### DISCUSSION TITLE
Dropping message with invalid payload

### DISCUSSION THREAD

**Discussion by fabnavigator** [zwave-js/node-red-contrib-zwave-js#239] [Q&A]
I have my log set for warning and see a number of these messages:
```
2022-07-19T15:21:02.530Z DRIVER   Dropping message with invalid payload
```
Should I be concerned?

**Comment by marcus-j-davies** [comment]
This is just driver related noise.

The driver is extremely for sticking to specification (rightfully so), so will not accept messages that do not conform to specification.
There are compatibility flags for certain behaviours. but honestly, if you don't see any issues, it's just white noise.

**Reply by fabnavigator** [reply]
@marcus-j-davies I understand. It would just be nice if it told me which node it was.
